Quaver.Shared/Skinning/Menus/SkinMenuResults.cs

```diff
@@ -0,0 +1,68 @@
+using IniFileParser.Model;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Quaver.Shared.Config;
+
+namespace Quaver.Shared.Skinning.Menus
+{
+    public class SkinMenuResults : SkinMenu
+    {
+        public Texture2D ResultsAvatarBorder { get; private set; }
+        public Texture2D ResultsAvatarMask { get; private set; }
+        public Texture2D ResultsBackgroundFilter { get; private set; }
+        public Texture2D ResultsTabSelectorBackground { get; private set; }
+        public Texture2D ResultsLabelAccuracy { get; private set; }
+        public Texture2D ResultsLabelMaxCombo { get; private set; }
+        public Texture2D ResultsLabelPerformanceRating { get; private set; }
+        public Texture2D ResultsLabelRankedAccuracy { get; private set; }
+        public Texture2D ResultsLabelTotalScore { get; private set; }
+        public Texture2D ResultsLabelScore { get; private set; }
+        public Texture2D ResultsLabelBlueTeam { get; private set; }
+        public Texture2D ResultsLabelRedTeam { get; private set; }
+        public Texture2D ResultsMultiplayerTeamPanel { get; private set; }
+        public Texture2D ResultsScoreContainerPanel { get; private set; }
+        public Texture2D ResultsGraphContainerPanel { get; private set; }
+        public Texture2D ResultsMultiplayerFFAPanel { get; private set; }
+        public Texture2D ResultsBackground { get; private set; }
+        public string ResultsBackgroundType { get; private set; }
```

Comment:
You're going to want to use an enum here rather than a string. [ConfigHelper.ReadEnum()](https://github.com/Quaver/Quaver/blob/ui-redesign/Quaver.Shared/Config/ConfigHelper.cs#L271) will help with parsing.